### PR TITLE
Gordeenko/task2

### PR DESCRIPTION
**Pull request by IrinaGordeenko**
implement Task2

### CODE REVIEW

File: Tasks/Gordeenko/Task2/Task2.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import datetime
+time = datetime.datetime.now()
+
+x = int(input("Please choose an option: automatic (0) or user (1) time entry:"))
+
+if x == 0:
+    print(f"{time.hour} : {time.minute}")
+if x == 1:
+    time_aut_hour = int(input("Please enter the time in hour and minute format (hh):"))
+    time_aut_minute = int(input("Please enter the time in hour and minute format (mm):"))
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Gordeenko/Task2/Task2.py:12]: it should be a single input in the "hh:mm" format

File: Tasks/Gordeenko/Task2/Task2.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+# (часы <12 и 21-23  / минуты 45-59)
+#hour = d.get(h)
+#hour_2 = d_2.get(h)
+#h2 = (int(h+1))
+#hour_3 = d.get(h2)
+#m3 = (int(60-m))
+#minute_3 = d_8.get(m3)
+#minute_4 = d_7.get(m3)
+#print(f"без {minute_3[0]} минут{minute_4[0]} {hour_3[0]}")
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Gordeenko/Task2/Task2.py:96]: why it's commented? should I evaluate it or not?

**Review by MikitaTsiarentsyeu — CHANGES_REQUESTED** [review]
Please uncomment all the meaningful code, which you want me to evaluate

**Comment by IrinaGordeenko** [comment]
They were independent modules. They worked. Each separately.

Now a single code. Added only "if/elif/else"
Fixed a bug in the dictionary "d_5"

File: Tasks/Gordeenko/Task2/Task2.py (hunk 1 above)
Comment (IrinaGordeenko) [reply Tasks/Gordeenko/Task2/Task2.py]: Update single input in the "hh:mm" format

File: Tasks/Gordeenko/Task2/Task2.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+d_5 = {1:["первого"],2:["второго"],3:["третьего"],4:["четвертого"],5:["пятого"],6:["шестого"],7:["седьмого"],8:["восьмого"],9:["девятого"],10:["десятого"],11:["одиннадцатого"],12:["двенадцатого"],21:["девятого"],22:["десятого"],23:["одиннадцатого"],24:["двенадцатого"]}
+d_6 = {1:["а"],2:["ы"],3:["ы"],4:["ы"],5:[""],6:[""],7:[""],8:[""],9:[""],10:[""],11:[""],12:[""],13:[""],14:[""],15:[""],16:[""],17:[""],18:[""],19:[""],20:[""],21:["а"],22:["ы"],23:["ы"],24:["ы"],25:[""],26:[""],27:[""],28:[""],29:[""],30:[""],31:["а"],32:["ы"],33:["ы"],34:["ы"],35:[""],36:[""],37:[""],38:[""],39:[""],40:[""],11:["а"],42:["ы"],43:["ы"],44:["ы"]}
+d_7 = {1:["ы"],2:[""],3:[""],4:[""],5:[""],6:[""],7:[""],8:[""],9:[""],10:[""],11:[""],12:[""],13:[""],14:[""],15:[""]}
+d_8 = {1:["одной"],2:["двух"],3:["трех"],4:["четырех"],5:["пяти"],6:["шести"],7:["семи"],8:["восьми"],9:["девяти"],10:["десяти"],11:["одиннадцати"],12:["двенадцати"],13:["тринадцати"],14:["четырнадцати"],15:["пятнадцати"]}
+
+
+if x == 0: # automatic
+    import datetime
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Gordeenko/Task2/Task2.py:16]: it's better to place all imports at the beginning of the file

File: Tasks/Gordeenko/Task2/Task2.py [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    elif ((h>=0 and h<=11) or (h>=21 and h<=24)) and (m>=45 and m<=59): # (часы <12 и 21-23  / минуты 45-59)
+        hour = d.get(h)
+        hour_2 = d_2.get(h)
+        h2 = (int(h+1))
+        hour_3 = d.get(h2)
+        m3 = (int(60-m))
+        minute_3 = d_8.get(m3)
+        minute_4 = d_7.get(m3)
+        print(f"без {minute_3[0]} минут{minute_4[0]} {hour_3[0]}")
+    else: print("error")
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Gordeenko/Task2/Task2.py:137]: it's a copy of the logic above, you could initially set up variables for hours and minutes (either from user or from datetime) first and then describe your logic only once

**Review by MikitaTsiarentsyeu — APPROVED** [review]
It's not very optimal, the data structure is overcomplicated, as well as the main logic, but still it's an okay solution. The score is 8.5.

**Pull request merged**